core/src/main/java/com/datastax/oss/driver/internal/core/channel/ProtocolInitHandler.java

```diff
@@ -150,29 +153,36 @@ protected boolean setConnectSuccess() {
 
     @Override
     String describe() {
-      return "[" + logPrefix + "] init query " + step;
+      return String.format(
+          "[%s] Protocol initialization request, step %d (%s)", logPrefix, stepNumber, request);
```

Comment:
For the record, `AuthResponse#toString()` doesn't include the token, so no risk of leaking sensitive information in the logs.